pkgs/top-level/aliases.nix

```diff
@@ -971,6 +971,12 @@ mapAliases ({
 
   posix_man_pages = man-pages-posix; # Added 2021-04-15
 
+  /* Cleanup before 21.11 */
+  avian = throw ''
+    The package doesn't compile anymore on NixOS and both development
+    & maintenance is abandoned by upstream.
+  '';
```

Comment:
do you mind adding the `# Added 2021-05-07`.

May help in the future to garbage-collect entries no one cares about in the future

Reply:
@jonringer done.